### PR TITLE
Upgrade Go to 1.16

### PR DESCRIPTION
**Pull request by antonioiubatti93**

### CODE REVIEW

**Comment by antonioiubatti93** [comment]
[Upgrade Go to 1.16](https://trello.com/c/ULgz3yER/1355-upgrade-go-to-116)

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.13.x, 1.14.x, 1.15.x]
+        go-version: [1.13.x, 1.14.x, 1.15.x, 1.16.x]
```
Comment (caroline105) [inline .github/workflows/test.yml:7]: Why are there several versions?

Comment (antonioiubatti93) [reply .github/workflows/test.yml:7]: because these are the versions the repo is meant to support.

**Comment by antonioiubatti93** [comment]
My bad, please look at #23

**Pull request closed without merging**